### ISSUE TITLE
scanpy.external.pp.dca results still need normolize,scale,and log1p ?

### ISSUE THREAD

**Issue by rdf1993** [theislab/dca#67]
If I run scanpy.external.pp.dca with default parameters, do I still need to perform normalization, scaling, and log1p afterward? How can I obtain the non-normalized values?

scanpy.external.pp.dca(adata, mode='denoise', ae_type='nb-conddisp', normalize_per_cell=True, scale=True, log1p=True, hidden_size=(64, 32, 64), hidden_dropout=0.0, batchnorm=True, activation='relu', init='glorot_uniform', network_kwds=mappingproxy({}), epochs=300, reduce_lr=10, early_stop=15, batch_size=32, optimizer='RMSprop', random_state=0, threads=None, learning_rate=None, verbose=False, training_kwds=mappingproxy({}), return_model=False, return_info=False, copy=False)[](https://scanpy.readthedocs.io/en/stable/generated/scanpy.external.pp.dca.html#scanpy.external.pp.dca)
